lib/src/filters/bassboost_filter.dart

```diff
@@ -23,19 +23,12 @@ enum BassBoostEnum {
       };
 }
 
-abstract class BassBoostInternal {
-  const BassBoostInternal(SoundHash? soundHash) : _soundHash = soundHash;
+abstract class BassBoostInternal extends FilterBase {
```

Comment:
[nit] If these `SomeFilterInternal` classes really are supposed to be internal, then I'd suggest:

1. Making the classes private (by renaming to `_BassBoostInternal`), OR
2. If they cannot be made private (i.e. only accessible within the same file), then maybe mark them as `@internal` or `@visibleForTesting`

If they're more like Base classes (publicly visible), then they could be renamed to something like `BassBoostBase`.

Reply:
done.